Appraisals

```diff
@@ -8,27 +8,39 @@ end
 
 appraise 'rails-5.0-stable' do
   gem 'rails', '~> 5.0.0'
+  group :test do
+    gem 'rails-controller-testing'
```

Comment:
What was this gem added for? This is an outdated method of testing and generally meant to support legacy apps, not for new code.

Reply:
We have some tests that need to run against controllers in Rails 4.2.  This lets them work against rails 5+ as well.